R/version.R

```diff
@@ -68,18 +68,22 @@ renv_version_match <- function(versions, request) {
 
 }
 
-renv_version_components <- function(version, n) {
```

Comment:
Why not keep `renv_version_components` and just have a helper `renv_version_majmin()` based on that?

Reply:
Because it wasn't used anywhere else? 

I think we have some difference in philosophy when a function is no longer used, because I just want to get rid of it 😄

Reply:
Here's an important question: do you have a giant box full of old cables somewhere in your house? 🙂 

(you can probably guess what my answer is)

Reply:
I have a box that is organised by cable type and whenever I get more than five of one type, I throw the worst out 🤣